neon-animated-pages.html

```diff
@@ -86,19 +86,17 @@
 
     },
 
-    observers: [
-      '_selectedChanged(selected)'
-    ],
-
     listeners: {
+      'iron-select': '_onIronSelect',
       'neon-animation-finish': '_onNeonAnimationFinish'
     },
 
-    _selectedChanged: function(selected) {
-
-      var selectedPage = this.selectedItem;
+    _onIronSelect: function(event) {
```

Comment:
I did a quick check to see if `neon-animated-pages` supported `null` or `undefined` as a selection value. It looks like it does not, so this revised implementation should be fine.